.github/actions/setup-solana/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup Solana CLI
```

Comment:
I think there is room here to move this whole process to a common action for multiple reasons.
1. We do the same script to install the solana cli in both repositories.
2. The chainlink repo script is out of date with the solana script so we need a single source of truth. chainlink has 1.9.12 while chainlink-solana has 1.10.33.